selfdrive/car/docs_definitions.py

```diff
@@ -325,9 +332,8 @@ def get_column(self, column: Column, star_icon: str, video_icon: str, footnote_t
     elif column == Column.VIDEO and len(item) > 0:
       item = video_icon.format(item)
 
-    footnotes = get_footnotes(self.footnotes, column)
-    if len(footnotes):
-      sups = sorted([self.all_footnotes[fn] for fn in footnotes])
-      item += footnote_tag.format(f'{",".join(map(str, sups))}')
+    # hardware footnotes need special formatting
+    if column != Column.HARDWARE:
+      item += self.get_ordered_footnotes(footnote_tag, column)
```

Comment:
Hmm, I'd like if we could keep this as generic as possible and still add the footnotes after. Perhaps this get_column can instead do the markdown spoiler wrapping generically? Would that be cleaner?